types/markdown-it/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "highlight.js": "^10.1.0"
```

Comment:
_ditto_
here DT is at v10.0, which is based on removed version:
https://github.com/markdown-it/markdown-it/blob/10.0.0/package.json#L43
it just happens there is no compiler errors.